### PR TITLE
Task 15.1 Create endpoint for getting information about current workspace

### PR DESCRIPTION
**Pull request by DoctorHeisenberg**

### CODE REVIEW

File: Provis.Core/Services/WorkspaceService.cs [hunk 1]
```diff
@@ -215,7 +215,28 @@ public async Task AcceptInviteAsync(int inviteId, string userid)
...
+
+        public async Task<WorkspaceInfoDTO> GetWorkspaceInfoAsync(int workspid, string userid)
+        {
+            var user = await _userManager.FindByIdAsync(userid);
+            
+            if (user == null)
+            {
+                throw new HttpException(System.Net.HttpStatusCode.NotFound, "User with Id not exist");
+            }
+            var userworkspace = _userWorkspaceRepository.Query().Where(x => x.WorkspaceId == workspid && x.UserId == userid)
```
Comment (YevheniiVovk) [inline Provis.Core/Services/WorkspaceService.cs]: Naming workspId same with userId

File: Provis.Core/Services/WorkspaceService.cs [hunk 2]
```diff
@@ -215,7 +215,28 @@ public async Task AcceptInviteAsync(int inviteId, string userid)
...
+                throw new HttpException(System.Net.HttpStatusCode.NotFound, "Workspace with this Id doesn't exist or you hasn't permissions");
+            }
+
+            var workspace = _mapper.Map<WorkspaceInfoDTO>(userworkspace);
+
+            return workspace;
         }
     }
-}
+}
```
Comment (YevheniiVovk) [inline Provis.Core/Services/WorkspaceService.cs]: Empty line

File: Provis.Core/Interfaces/Services/IWorkspaceService.cs [hunk 3]
```diff
@@ -16,5 +16,7 @@ public interface IWorkspaceService
         Task<List<WorkspaceInfoDTO>> GetWorkspaceListAsync(string userid);
 
         Task SendInviteAsync(InviteUserDTO inviteUser, string userId);
+
+        Task<WorkspaceInfoDTO> GetWorkspaceInfoAsync(int id, string userid);
```
Comment (YevheniiVovk) [inline Provis.Core/Interfaces/Services/IWorkspaceService.cs]: Same with naming here

File: Provis.Core/Services/WorkspaceService.cs [hunk 4]
```diff
@@ -215,7 +215,29 @@ public async Task AcceptInviteAsync(int inviteId, string userid)
...
+            
+            if (user == null)
+            {
+                throw new HttpException(System.Net.HttpStatusCode.NotFound, "User with Id not exist");
+            }
+
+            var userworkspace = _userWorkspaceRepository.Query().Where(x => x.WorkspaceId == workspId && x.UserId == userId)
+                .Include(x => x.Workspace).Include(x => x.Role).FirstOrDefault();
+
+            if (userworkspace == null)
```
Comment (YevheniiVovk) [inline Provis.Core/Services/WorkspaceService.cs]: naming userworkspace

File: Provis.Core/Services/WorkspaceService.cs [hunk 5]
```diff
@@ -215,7 +215,29 @@ public async Task AcceptInviteAsync(int inviteId, string userid)
...
+            {
+                throw new HttpException(System.Net.HttpStatusCode.NotFound, "Workspace with this Id doesn't exist or you hasn't permissions");
```
Comment (YevheniiVovk) [inline Provis.Core/Services/WorkspaceService.cs]: Length should be less 100 characters please split it by ,

**Review by YevheniiVovk — APPROVED** [review]

**Review by AndriyKovalets — APPROVED** [review]

**Review by Artem672 — APPROVED** [review]

**Pull request merged**